datasets/my/aob_sanctions/my_aob_sanctions.yaml

```
@@ -0,0 +1,230 @@
+title: Malaysian's Securities Comission Sanctions
+entry_point: crawler.py
+prefix: my-aob
+coverage:
+  frequency: daily
+  start: 2024-04-12
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Auditors sanctioned by the Malaysian's Securities Comission Audit Oversight Board
+description: |
+  This dataset comprises information about auditors sanctioned by
+  the Malaysian's Securities Comission Audit Oversight Board,
+  The focus of AOB enforcement is whether the auditors comply
+  with the recognised auditing and ethical standards.
+  Such action from the AOB may not necessary imply the audited
+  financial statement does not give a true and fair view.
+  The sanctions are: 
+    - Directing the person in breach to comply with the
```

This gets interpreted as markdown, so it would be great to get blank lines to split the text into paragraphs to make reading a bit easier.
```suggestion
description: |
  This dataset comprises information about auditors sanctioned by
  the Malaysian's Securities Comission Audit Oversight Board.
  
  The focus of AOB enforcement is whether the auditors comply
  with the recognised auditing and ethical standards.
  Such action from the AOB may not necessary imply the audited
  financial statement does not give a true and fair view.
  
  Potential sanctions are: 
  
    - Directing the person in breach to comply with the
```